src/main.rs

```diff
@@ -104,6 +104,7 @@ fn handle_conversion(input: &str, output_dir_path: &String, html_lang: &str) {
 fn create_output_directory(output_dir_path: &String) {
   // Delete output dir and its contents if it is the default output dir
   if output_dir_path == DEFAULT_OUTPUT_DIR && path::Path::new(DEFAULT_OUTPUT_DIR).exists() {
+    println!("REMOVING OLD DIRECTORY");
```

Comment:
Could you please remove this print statement?

Reply:
Sorry, my bad